src/plugins/formvalid/formvalid.js

```diff
@@ -319,8 +319,16 @@ var componentName = "wb-frmvld",
 											// Create our container if one doesn't already exist
 											if ( $summaryContainer.length === 0 ) {
 												$summaryContainer = $( "<section id='" + errorFormId + "' class='alert alert-danger' tabindex='-1'>" + summary + "</section>" ).prependTo( $form );
-											} else {
+											} else if ( $summaryContainer.html().replace( /'/g, "\"" ).replace( /&#160;/g, "&nbsp;" ) !== summary.replace( /'/g, "\"" ).replace( /&#160;/g, "&nbsp;" ) ) {
+
+												// if the summary container is currently focused then
+												// we will focus the last element in the summary
+												// since we recreate the error list on blur/change
+												var $isFocused = $summaryContainer.find( "a" ).is( ":focus" );
 												$summaryContainer.empty().append( summary );
+												if ( $isFocused ) {
+													$summaryContainer.find( "a" ).last().trigger( "focus" );
+												}
 											}
```

Comment:
- Suggested comment
- Small code optimization
- Added spacing for code readability.

```suggestion

											// Replace current error summary content only if it is different 
											// from the incoming error summary content
											} else if ( $summaryContainer.html() !== summary.replace( /'/g, "\"" ).replace( /&#160;/g, "&nbsp;" ) ) {
												$summaryContainer.empty().append( summary );

												// if the summary container is currently focused then
												// we will focus the last element in the summary
												// since we recreate the error list on blur/change
												if ( $summaryContainer.find( "a" ).is( ":focus" ) ) {
													$summaryContainer.find( "a" ).last().trigger( "focus" );
												}
											}
```